### ISSUE TITLE
AttributeError: 'PeriodicVestingAccount' object has no attribute 'account_number'

### ISSUE THREAD

**Issue by geemaple** [terra-money/terra.py#155]
- terra_sdk version: 3.0.0
- Python version: 3.9.1
- Operating System: MacOS 12.4

### Description

I'm tring to create a CreateTxOptions and sign it with my wallet using this sdk

### What I Did

```
  File "/Users/dean/Developer/luna_testing.py", line 126, in swap
    tx = wallet.create_and_sign_tx(options=tx_options)
  File "/Users/dean/.virtualenvs/luna/lib/python3.9/site-packages/terra_sdk/client/lcd/wallet.py", line 123, in create_and_sign_tx
    res = self.account_number_and_sequence()
  File "/Users/dean/.virtualenvs/luna/lib/python3.9/site-packages/terra_sdk/client/lcd/wallet.py", line 85, in account_number_and_sequence
    return {"account_number": res.account_number, "sequence": res.sequence}
AttributeError: 'PeriodicVestingAccount' object has no attribute 'account_number'
```

**Comment by JSHan94** [comment]
This is related with #153 . will be fixed soon. thanks you for your report.

**Comment by JSHan94** [comment]
Closes #159
